lib/data/receiver.js

```diff
@@ -55,6 +60,7 @@ class DataTrackReceiver extends DataTrackTransceiver {
 
 /**
  * @event DataTrackReceiver#message
+ * @event DataTrackReceiver#close
```

Comment:
@makarandp0 "close" should be a separate block and should not be a part of this block.